util/src/test/java/org/killbill/billing/mock/api/MockAccountUserApi.java

```diff
@@ -167,4 +167,10 @@ public void updateAccount(final UUID accountId, final AccountData accountData, f
             throws AccountApiException {
         throw new UnsupportedOperationException();
     }
+
+    @Override
+    public List<Account> getChildrenAccounts(final UUID uuid, final TenantContext tenantContext) throws AccountApiException {
+        // TODO implement
```

Comment:
If you don't need it for now, could you `throw new UnsupportedOperationException();`?